### PR TITLE
Issue 72 and 73

### LINKED ISSUE #72: GET /posts/{post_id}/comments
get comments for a single post

### LINKED ISSUE #73: POST /posts/{post_id}/comments
Add a comment to a post

### PR DESCRIPTION
**Pull request by ColeBoytinck**
Closes #72

### CODE REVIEW

File: web_server/posts/views.py [hunk 1]
```diff
@@ -141,13 +141,49 @@ def html_handler(request, posts, pager, pagination_uris):
...
+            output['next'] = next_uri
+
+        return JsonResponse(output)
+
     if request.method == 'POST':
         # JSON post body of what you post to a posts' comemnts
         # POST to http://service/posts/{POST_ID}/comments
         return HttpResponse("http://service/posts/{POST_ID}/comments POST")
     elif request.method == 'GET':
         # access to the comments in a post
```
Comment (AustinGrey) [inline web_server/posts/views.py]: This logic is not required if you are using the endpoint handler to determine which handler will service the request.

**Review by AustinGrey — CHANGES_REQUESTED** [review]
Please remove superfluous if...elif.
Also please note that #108 is getting merged prior to this PR, make sure your code is compatible.
@phoebebai also has reported that she has already created the commenting user interfaces, you should work with her to ensure that your endpoint is linked to her interfaces

File: web_server/posts/views.py (hunk 1 above)
Comment (ColeBoytinck) [reply web_server/posts/views.py]: Changed the logic

**Comment by ColeBoytinck** [comment]
both get and post have been completed for comments. This will not work with the front end ATM because of a spec issue

**Review by AustinGrey — APPROVED** [review]

**Comment by phoebebai** [comment]
Hey, I am working on the front-end. I have a question about commenting: are users allowed to delete their comments?

**Comment by phoebebai** [comment]
@ColeBoytinck 
Could you please merge master into your branch first? Since Ida changed author model, I am thinking whether there will be bugs or conflicts. 
I saw "published": "2020-03-05T22:08:02.810Z". Could you please change it into more readable format and make sure it is MT ?

**Comment by ColeBoytinck** [comment]
Adding another API to allow for deleting comments,

In terms of the published format, that is a frontend time, and it is that way in the spec

**Pull request merged**